data/gui/window/help_browser.cfg

```diff
@@ -81,8 +81,11 @@
 	[resolution]
 		definition = "default"
 
+#ifndef IPHONEOS
 		{GUI_WINDOW_FIXED_SIZE_CENTERED 1200 700}
-
+#else
+		{GUI_WINDOW_FULLSCREEN}
+#endif
```

Comment:
Ultimately I'd prefer to see an entirely separate `[gui]` definition for iOS. These ifdefs are fine for the time being, but I imagine you may also want to redesign some dialogs quite a bit. (Note that having a separate `[gui]` doesn't mean you need to duplicate every dialog; it'll fall back to the default definitions if necessary.)